test_scripts/Polices/Validation_of_PolicyTables/preloaded_pt_validation_rules_for_required_parameters.lua

```diff
@@ -0,0 +1,113 @@
+---------------------------------------------------------------------------------------------
+-- Description:
+-- Behavior of SDL during start SDL in case when PreloadedPT has no required parameters
+-- 1. Used preconditions:
+-- Delete files and policy table from previous ignition cycle if any
+-- Do not start default SDL
+-- 2. Performed steps:
+-- Create PreloadedPolicyTable file without one required parameter
+-- Start SDL
+
+-- Requirement summary:
+-- [Policies]: PreloadedPolicyTable: Validation rules for required parameters
+--
+-- Expected result:
+-- SDL shuted down
+---------------------------------------------------------------------------------------------
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+local config = require('config')
+
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+local preloaded_pt_file_name = "sdl_preloaded_pt.json"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local json = require("modules/json")
+local SDL = require('modules/SDL')
+
+--[[ Preconditions ]]
+function Test.backup_preloaded_pt()
+  os.execute("cp " .. config.pathToSDL .. preloaded_pt_file_name .. ' ' .. config.pathToSDL .. "backup_" .. preloaded_pt_file_name)
+end
+
+function Test.restore_preloaded_pt()
+  os.execute("mv " .. config.pathToSDL .. "backup_" .. preloaded_pt_file_name .. " " .. config.pathToSDL .. preloaded_pt_file_name)
+end
+
+function Test.corrupt_preloaded_pt()
+  local changed_parameters = {
+    ["1234"] = {
+      [123] = "http://cloud.ford.com/global",
+      keep_context = false,
+      steal_focus = false,
+      -- priority = "NONE", -- removed required parameter
+      default_hmi = "NONE",
+      groups = {"BaseBeforeDataConsent"}
+    },
+    super = {
+      keep_context = false,
+      steal_focus = false,
+      priority = "NONE",
+      default_hmi = "NONE",
+      groups = {"BaseBeforeDataConsent"}
+    }
+  }
+
+  local pathToFile = config.pathToSDL .. preloaded_pt_file_name
+
+  local file = io.open(pathToFile, "r")
+  local json_data = file:read("*a")
+  file:close()
+
+  local data = json.decode(json_data)
+  if data then
+    for key, value in pairs(changed_parameters) do
+      data.policy_table.app_policies[key] = value
+    end
+  end
+
+  local dataToWrite = json.encode(data)
+  file = io.open(pathToFile, "w")
+  file:write(dataToWrite)
+  file:close()
+end
+
+function Test.check_sdl()
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnSDLClose",{}):Times(1)
+  local status = SDL:CheckStatusSDL()
+  if status == SDL.RUNNING then
+    commonFunctions:userPrint(31, "Test failed: SDL running without required parameters in preloaded policy table")
+    return false
+  end
+  return true
+end
+
+function Test:Precondition_stop_sdl()
+  StopSDL(self)
+end
+
+function Test:Precondition()
```

Comment:
Please check comment above about name of functions in section Preconditions.
Also please make name more descriptive in this case